src/EditorFeatures/TestUtilities2/Intellisense/OldCompletionTestState.vb

```diff
@@ -0,0 +1,438 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Completion
+Imports Microsoft.CodeAnalysis.Editor.CommandHandlers
+Imports Microsoft.CodeAnalysis.Editor.Implementation.Formatting
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
+Imports Microsoft.CodeAnalysis.SignatureHelp
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.VisualStudio.Commanding
+Imports Microsoft.VisualStudio.Language.Intellisense
+Imports Microsoft.VisualStudio.Text
+Imports Microsoft.VisualStudio.Text.Editor
+Imports Microsoft.VisualStudio.Text.Editor.Commanding.Commands
+Imports Roslyn.Utilities
+Imports CompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem
+Imports VSCommanding = Microsoft.VisualStudio.Commanding
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
+
+    ''' <summary>
+    ''' This is a TestState corresponding to the old Completion service which we are going to turn off after migration to the new one.
+    ''' We should keep this file up-to-date and on par with the new file until then.
+    ''' </summary>
+    Partial Friend Class OldCompletionTestState
```

Comment:
❔ Why is this not showing up as a simple file rename?

Reply:
We keep the original file for the new completion. Is it possible to split the file into two copies in Git?

---
In reply to: [208002486](https://github.com/dotnet/roslyn/pull/29016#discussion_r208002486) [](ancestors = 208002486)